### PR TITLE
Bower added to track html dependencies versions.

### PR DESCRIPTION
**Pull request by filipstachura**

### CODE REVIEW

File: inst/www/bower.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "shinyrouter",
+  "authors": [
+    "Filip Stachura <filip@appsilon.pl>"
```
Comment (KrzysztofWrobel) [inline inst/www/bower.json]: new email

**Comment by marekrogala** [comment]
- @filipstachura could you elaborate a bit on why we need that?
- does this need a README change? E.g. is this a change to how this package is built?

**Comment by filipstachura** [comment]
This way we can manage js dependencies in more convenient way (each is explicitly declared with version stated in `bower.json`). Bower is needed only to bump the version of dependency or installing a new one.

Why we need that?
- each package has version stated in bower.json
- easier bumping and installing of new deps

**Comment by filipstachura** [comment]
BTW We are still working on README and gh-page and we want to add a note about it in 'how to contribute'.

**Comment by marekrogala** [comment]
thanks for the info
ideally, we should have a rule that README update (just saying how to build the bower deps) would be included in the PR itself -- but of course I understand that it's hard when we have the "tech debt" of missing README

**Pull request merged**